docker-compose.yml

```diff
@@ -1,38 +1,64 @@
 version: '3.8'
 
 services:
-  nginx:
-    container_name: badge-poser-nginx
-    image: nginx:alpine
+  app:
```

Comment:
Sorry, but why you would use the docker-compose to development also to production?

Reply:
That would improve the chances of getting errors before going to production since the two environments will use exactly the same image.